openplotter.py

```diff
@@ -1829,7 +1860,6 @@ def apply_changes_outputs(self, e):
 		self.conf.set('OUTPUTS', 'outputs', str(self.outputs))
 		self.start_monitoring()
 		self.read_datastream()
-		self.read_actions()
```

Comment:
This line needs to be here. this is a solved bug and probably you forked the repository before was done.